presto-docs/src/main/sphinx/sql/create-function.rst

```diff
@@ -0,0 +1,83 @@
+===============
+CREATE FUNCTION
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    CREATE [ OR REPLACE ] FUNCTION
+    qualified_function_name (
+      parameter_name parameter_type
+      [, ...]
+    )
+    RETURNS return_type
+    [ COMMENT function_description ]
+    [ LANGUAGE language ]
+    [ DETERMINISTIC | NOT DETERMINISTIC ]
+    [ RETURNS NULL ON NULL INPUT | CALLED ON NULL INPUT ]
+    [ RETURN expression ]
+
+
+Description
+-----------
+
+Create a new function with the specified definition.
+
+Each function is uniquely identified by its qualified function name
+and its parameter type list. ``qualified_function_name`` needs to be in
+the format of ``catalog.schema.function_name``.
+
+To create a function, the function namespace as identified by
+``catalog.schema`` needs to managed by a
+:doc:`/admin/function-namespace-managers`.
+
+The optional ``OR REPLACE`` clause causes the ``ALREADY_EXISTS`` error
+to be suppressed if the function already exists.
+
+The ``return_type`` needs to match the actual type of the routine body
+``expression``, without performing type coercion.
```

Comment:
Match or assignment-compatible? For example, I can return 0 for a double returning function?

Reply:
This is not supported currently, but we're looking to support it in the future.